flytekit/extras/sqlite3/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.8-slim-buster
```

Comment:
will we keep this dockerfile here? Is the idea that some plugins can ship with their dockerfiles?

Reply:
I think i'm going to revert all these changes and add another example in the plugins folder.